server/routes/api/collections/collections.ts

```diff
@@ -862,9 +844,16 @@ router.post(
 
     collection.archivedAt = new Date();
     collection.archivedById = user.id;
-    await collection.save({ transaction });
     collection.archivedBy = user;
 
+    await collection.saveWithCtx(ctx, undefined, {
+      name: "archive",
+      data: {
```

Comment:
Are there processors relying on this? Could just change them to use `changes` – we don't need consistency of `data` with past events.

Comment applies to a couple of other spots too